src/components/common/Actions.vue

```diff
@@ -90,13 +93,15 @@ export default class Actions extends Vue {
   @Action setIsSavingResuming!: ActionBindingIF
   @Action setIsFilingPaying!: ActionBindingIF
 
+  // Local Properties
+  private authUrl = sessionStorage.getItem('AUTH_URL') || ''
```

Comment:
Please verify that this property gets the proper value, since getting AUTH_URL and putting in session storage is asynchronous with program startup. I'm concerned that the assignment above takes place before AUTH_URL session storage is set.